mantle/kola/tests/ignition/qemufailure.go

```diff
@@ -182,11 +182,8 @@ func ignitionFailure(c cluster.TestCluster) error {
 func dualBootfsFailure(c cluster.TestCluster) error {
 	builder := platform.NewQemuBuilder()
 	defer builder.Close()
-	// Create a temporary log file
-	consoleFile, err := builder.TempFile("console.log")
-	if err != nil {
-		return err
-	}
+	// Create a temporary log file allocated in the output dir of the test
+	consoleFile := c.H.TempFile("console.log")
```

Comment:
Can we do the same in the other tests in this file?

Also, the argument passed in is a prefix, which means that the actual file looks like e.g. `console.logRANDOMSTUFF`, which is awkward. So maybe let's just pass in `console-`?

Reply:
That's fair, indeed the random stuff did look awkward. As requested I changed it now for all of the tests in this file. Thanks for the suggestion.